packages/core/src/Box/README.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Box'
+type: 'component'
+slug: /components/box/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Box'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-box--default'
+---
+
+Box is the most abstract layout component that allows to apply `margin` and `padding` properties according to the [spacing system]. By default, it renders a `div` element.
```

Comment:
@suevalov thx a lot for those docs! 🎉
Small comment from my side, that maybe we could follow similar naming for sections in the documentation as we have for `Components` (of course not all of them would be relevant for Layout components, but maybe we could follow similar conversion). This is how they look like:

## Table of contents
- [How to use `compontent name`](#how-to-use-buttons)
- [Component variations](#component-variations)
- [Code examples](#code-examples)
- [Content recommendations](#content-recommendations)
- [Accessibility](#accessibility)

Reply:
What's the difference between `How to use...`, `Component variations` and `Code examples`? Aren't all of them about how to use components?

What is `Content recommendations` section?

Reply:
For example `Tag` component:

## How to use:
- tags are used to inform users of the status of an element in the interface
....
## Component variations:
 - **Primary** - used for the entity that has been changed
 ...

## Content recommendations
- try to use labels with short, scannable text
....

`Component variations` was supposed to show variants and describe those. `Code example` was thought as a section with examples of usage in code. Those 2 sections not necessarily are always present.

Reply:
Updated!